src/components/loader/loader-square.style.ts

```diff
@@ -48,29 +48,32 @@ const getDimensions = (size: StyledLoaderSquareProps["size"]) => {
   `;
 };
 
+export const StyledLoaderPlaceholder = styled.div<
+  Pick<StyledLoaderSquareProps, "size">
+>`
+  display: inline-block;
+  ${({ size }) => getDimensions(size)}
```

Comment:
comment: I had this wrong it needs to have a min-width of 64px to account for the `squares` when they actually render

https://gist.github.com/edleeks87/d899b4701a65f3157f020e37689f8ad1